content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
 ---
 title: "Access"
-description: "Accessing GOCDB"
-weight: 20
+description: "Accessing the configuration database"
```

Comment:
```suggestion
description: "Accessing the Configuration Database"
```

I would capitalise the service name everywhere (so Configuration Database)

Reply:
I agree, in principle. But not everywhere the text is referring to the official service (with its official name). The term "configuration database" also makes sense on its own, when describing what it actually is or when discussing it. But in this case, yes, you are right.